src/BitOasis/Bitfinex/ExchangeRate/RabbitMqExchangeRateUpdater.php

```diff
@@ -19,12 +19,16 @@
 class RabbitMqExchangeRateUpdater implements LoggerAwareInterface {
 	use LoggerAwareTrait;
 
-	const WEB_RPC_EXCHANGE = 'bitoasis.web-events';
+	const WEB_RPC_EXCHANGE = 'bitoasis.fiat-rates';
 	const WEB_RPC_EXCHANGE_TYPE = 'direct';
+	const WEB_RPC_QUEUE_NAME = 'usd-exchange-rate.update';
 
 	/** @var int|null */
 	protected $prefetchCount = 25;
 
+	/** @var string */
+	protected $queuePostfix = '';
```

Comment:
Why is this needed?

Reply:
If the queue will be same for both orderbook and trade, only one of them will consume message from queue :) So added a postfix to create different queues for different microservices. 

As of now, orderbook will create queue `usd-exchange-rate.update.orderbook` and trades will create `usd-exchange-rate.update.trades`. One message => 2 queues => 2 consumers